application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -133,7 +135,13 @@ public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
         // Deleted messages cause retrieveMessageById to fail.
         Consumer<Message> notHandledAction =
                 any -> transferFlow(event, channelId, authorId, messageId);
-        Consumer<Throwable> handledAction = any -> alreadyHandled(sourceChannel, helperForum);
+
+        Consumer<Throwable> handledAction = any -> {
+            if (any instanceof ErrorResponseException errorResponseException
+                    && errorResponseException.getErrorResponse() == ErrorResponse.UNKNOWN_MESSAGE) {
+                alreadyHandled(sourceChannel, helperForum);
+            }
```

Comment:
... and `if` not, perhaps log the issue as error. `log.error("Unknown error while ...", any)`.
And since u now actually use `any`, it shouldnt be named like that anymore. Perhaps `error` or `failure`.